src/App.tsx

```diff
@@ -10,33 +12,20 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTab, setSelectedTab] = useState<Tab>(tabs[0]);
```

Comment:
TS already knows about this type
```suggestion
  const [selectedTab, setSelectedTab] = useState(tabs[0]);
```